### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by KwinyarutP**

### CODE REVIEW

File: client/src/pages/home/HomePage.jsx [hunk 1]
```diff
@@ -1,13 +1,58 @@
-import React from "react";
 import { useState, useEffect } from "react";
-import Navbaruser from "../../components/Navbaruser";
+// import Navbaruser from '../../components/Navbaruser';
```
Comment (Chatchaponz) [inline client/src/pages/home/HomePage.jsx:2]: Remove unused code na

File: client/src/pages/home/HomePage.jsx [hunk 2]
```diff
@@ -1,13 +1,58 @@
...
+import SearchBar from '../../components/Searchbar';
+import CustomBtn from '../../components/Botton';
+import api from '../../hooks/api';
 
+// client/src/components/Navbaruser.jsx
```
Comment (Chatchaponz) [inline client/src/pages/home/HomePage.jsx:7]: Remove unused code na

File: client/src/pages/home/HomePage.jsx [hunk 3]
```diff
@@ -1,13 +1,58 @@
...
 export const Home = () => {
   const [message, setMessage] = useState("");
+  const [searchResults, setSearchResults] = useState([]); // hold search results
+  const [selectedSearch, setSelectedSearch] = useState(null); // hold the selected item
+  const [query, setQuery] = useState(''); // hold the search query
+
+  useEffect(() => {
+    const fetchSearchResults = async () => {
+      try {
+        const response = await api.get("/search"); // Replace with your API endpoint
```
Comment (Chatchaponz) [inline client/src/pages/home/HomePage.jsx:17]: Do u already test this krub?

Comment (KwinyarutP) [reply client/src/pages/home/HomePage.jsx:17]: Screenshot 2567-08-10 at 00 19 21
is it correct ka?

**Pull request merged**